node-services/cards-external-diffusion/src/domain/client-side/cardsExternalDiffusionService.ts

```diff
@@ -70,18 +92,48 @@ export default class CardsExternalDiffusionService {
         this.active = false;
     }
 
-    public isActive() : boolean {
+    public isActive(): boolean {
         return this.active;
     }
 
     private checkRegularly() {
         if (this.active) {
             this.logger.info('Check regularly');
-            this.cardsDiffusionControl
+            this.realTimeCardsDiffusionControl
                 .checkUnreadCards()
                 .catch((error) => this.logger.error('error during periodic check' + error))
                 .finally(() => setTimeout(() => this.checkRegularly(), this.checkPeriodInSeconds * 1000));
+        } else setTimeout(() => this.checkRegularly(), this.checkPeriodInSeconds * 1000);
+    }
+
+    private checkDaily() {
+        if (this.active) {
+            this.logger.info('Daily check');
+            const now = new Date();
+            const configHour = new Date(
+                now.getFullYear(),
+                now.getMonth(),
+                now.getDate(),
+                this.hourToSendDailyEmail,
+                this.minuteToSendDailyEmail,
+                0,
+                0
+            );
+            let millisTillConfigHour = configHour.getTime() - now.getTime();
+            if (millisTillConfigHour < 0) {
+                millisTillConfigHour += 24 * 60 * 60 * 1000;
+            }
+
+            setTimeout(() => {
+                this.sendDailyRecap();
+                setInterval(() => {
+                    this.dailyCardsDiffusionControl.checkCardsOfTheDay();
+                }, 24 * 60 * 60 * 1000);
+            }, millisTillConfigHour);
         }
-        else setTimeout(() => this.checkRegularly(), this.checkPeriodInSeconds * 1000)
     }
-}
\ No newline at end of file
+
+    public async sendDailyRecap() {
+        this.dailyCardsDiffusionControl.checkCardsOfTheDay();
```

Comment:
add a log.info to say daily mail process start

add a try/ catch  to check potential error (falling to do so could crash the program)